llap-common/src/protobuf/LlapDaemonProtocol.proto

```diff
@@ -211,6 +211,14 @@ message PurgeCacheResponseProto {
   optional int64 purged_memory_bytes = 1;
 }
 
+message GetLoadMetricsRequestProto {
+}
+
+message GetLoadMetricsResponseProto {
+  optional int64 num_executors_available = 1;
```

Comment:
i would change this to a Map<String,int64> for numeric values and add map entries for each counter/metric rather than listing out each of them as separate  message values.

Reply:
Used a map like sturcutre since the map is not available for 2.5.0 protobuf we are using